hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -381,20 +340,18 @@ private void sendState() throws InterruptedException, MalformedURLException {
 
                 JsonObject root = new JsonObject();
                 root.add("identifier", identifier.toJson());
-                TimedMemberState memberState = timedMemberState.get();
-                if (memberState != null) {
```

Comment:
The returned TimedMemberState will never be null, so we can remove the null check